### ISSUE TITLE
delete / backspace OS X 

### ISSUE THREAD

**Issue by MacroMachines** [szymonkaliski/SDF-UI#5]
First off, incredible work, and great coincidense as I was just starting on something similar.  

I am working from a MacBook pro 17 inch made years back.  I can not seem to delete any nodes or connections.  I did a search on google and it seems the delete key should be the same as the PC backspace but it is not seeming to do the expected behavior of removing a node or connection.

Safari OSX 10.12.3

**Comment by MacroMachines** [comment]
appears to work in chrome though.

**Comment by szymonkaliski** [comment]
yeah, it's not tested in anything other than chrome, PRs are welcome, otherwise I don't think I'll get to it soon :) 
thanks for your notes though! 
what are you building? I could help possibly ;)

**Comment by MacroMachines** [comment]
I am building a few things based on GLSL / RayMarching / SDF, the most exciting of which is a unique modular synthesis environment that generates audio and visual elements together for realtime audio/visual performance.  I've done a fair bit of DSP/audio programming and built up my skill in that area to a sufficient degree, and have been honing my GLSL skill.  The plan is to have the performance controls and preview running on iPads, and performers can connect to a main desktop with a decent graphics card which does the final render to projection.  I plan to use OSC & MIDI for the communication of parameters / musical events.  Would love to tell you more if you are interested.  It looks like you have done some of the things I was beginning on, such as this node environment for creating the SDF forms.

**Comment by szymonkaliski** [comment]
Oh this looks fun and right up my alley, hit me at hi@szymonkaliski.com :)

**Comment by MacroMachines** [comment]
cool, emailing now :D
